justfile

```
@@ -0,0 +1,11 @@
+_default:
+  @just --list
+
+clean:
+  rm -r cosmic-settings cosmic-term | true
+
+build: clean
+  whiskers templates/cosmic-settings.tera
+  mv themes cosmic-settings
+  whiskers templates/cosmic-term.tera
+  mv themes cosmic-term
```

I'm not sure why we need the folders to be at the top level? I'd want this to be simplified to:

```suggestion
clean:
  rm -rf themes

build: clean
  whiskers templates/cosmic-settings.tera
  whiskers templates/cosmic-term.tera
```

---

The `| true` on the `rm` command makes sure that the justfile doesn't error if the themes directory didn't already exist. I'll make the other adjustments though.